packages/itwin/measure-tools/src/ui-2.0/MeasureToolsUiProvider.tsx

```diff
@@ -57,7 +57,7 @@ export class MeasureToolsUiItemsProvider implements UiItemsProvider {
         return [
           ToolbarItemUtilities.createGroupButton(
             "measure-tools-toolbar",
-            500,
+            1000000,
```

Comment:
Does this look right? If I remove a 0, the button moves back up to the top, but 1000000 seems like an unusually large number.

Reply:
@bsteinbk this doesnt seem right. Any idea?

Also, that debug tool you showed off a few months ago to show the item priority, is that available anywhere?

Reply:
@aruniverse I see that constructor takes a prop for itemPriority that seem like a  good idea, to future-proof the api you may also allow a groupPriority to be specified. 

As for the tool I demoed - it is only in ui-test-app. I will try to separate it out into a debugToolItemsProvider.